### DISCUSSION TITLE
Setting up Debian VM

### DISCUSSION THREAD

**Discussion by macaronibchan** [isc30/ryzen-gpu-passthrough-proxmox#8] [Q&A]
Hi guys, I followed the guide and was able to the iGPU to passthrough to Windows, thanks!

I tried replicating something similar on Debian 12 and Ubuntu and have no luck getting to work so far (short of downloading drivers from AMD). Does anyone know of a tutorial towards getting this working on a linux build?

**Comment by isc30** [comment]
Hey, I managed to get it working in Ubuntu without any special drivers, just the built-in ones. The trick is the on Ubuntu the remote desktop client doesn't support GPU acceleration, so it seems as if the GPU is "not working". But if you open a native tool that uses the GPU (like transcoding), it worked fine

**Reply by macaronibchan** [reply]
Can I ask how you went about creating the VM? Was it like Windows where you created the VM first then edited it to passthrough the iGPU, then apt updated?

I tried it with Ubuntu then installed Mesa but it didn't seem to work. Neofetch states the hex value of the iGPU so I'm not sure where I'm going wrong.

**Reply by isc30** [reply]
yeah, the VM creation and passthrough of the PCI device is exactly the same as SeaBIOS for windows 10. You can use commands like `lspci` from the debian VM to check if it detects the GPU properly etc

**Reply by macaronibchan** [reply]
Thanks for the followup. I tried it again with a fresh install and noticed that when I run' lspci -nn | grep VGA' I get two results
One called the VGA compatible controller and the other is the passthrough Phoenix APU? Do I need to disable the VGA controller and if so can you share how to go about it?

Otherwise with the above setup, when I go to run 'glgears' im only getting approximate 2.5k fps

Any help if much appreciated, thanks!
